core/domain/collection_services_test.py

```diff
@@ -1566,7 +1455,7 @@ class CollectionSearchTests(CollectionServicesUnitTests):
     """Test collection search."""
 
     def test_demo_collections_are_added_to_search_index(self):
-        results = collection_services.search_collections('Welcome', 2)[0]
+        results = collection_services.search_collections('Mathematics', 2)[0]
```

Comment:
Can this be changed back to Welcome? It seems to make the test a bit tighter, since the demo is part of the welcome category.